src/data-service/SequelizeConnection.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize');
+
+let sequelizeConnection = function () {
+  let connect = new Sequelize('DB-Battleship', 'sa', 'UserSA12',{
```

Comment:
FYI is a bad practice to push credentials to the code base

Reply:
The values were removed